src/Stratis.Bitcoin/Base/InvalidBlockHashStore.cs

```diff
@@ -60,7 +60,7 @@ public class InvalidBlockHashStore : IInvalidBlockHashStore
         /// All access to this object has to be protected by <see cref="lockObject"/>.
         /// <para>The field is internal for testing purposes.</para>
         /// </remarks>
-        internal readonly CircularArray<uint256> orderedHashList;
+        internal CircularArray<uint256> OrderedHashList { get; private set; }
```

Comment:
`We use camelCase for internal and private fields`
CodeStyle:3

Reply:
OK will make this private and fix the test.